packages/fxa-auth-server/lib/senders/emails/scss/appBadges.scss

```diff
@@ -0,0 +1,12 @@
+.app-badges img {
```

Comment:
> Kind of an aside note, but I think we can have a soft rule that partials don't necessarily need Storybook stories (anyone feel free to disagree with me) or tests if they're covered by templates.

I generally agree, but it would be nice to have stories for far-reaching partials, like buttons, and avatar, footer, etc - to allow for easier template building down the road. But I also don't want overkill here.

Reply:
Yeah the directory structure that you set out to me definitely looks way better than what I have currently. But I'm confused as to how we will be compiling those `.scss` files to `css` files. What I mean to say is currently `appBadges.scss` gets compiled to `appBadges.css` but with the structure that you've mentioned above, all the scss files have similar name i.e `index.scss` so are we planning on compiling all the scss files into one file say `main.css` or do you want me to create subdirectories inside the css folder. Something like this maybe:

```
templates/
  cadReminder/
    index.ts // hopefully we can change this to `.mjml` or `.ejs`?
    index.stories.tsx
    index.scss
    index.test.ts // can be done in that "convert cadReminder" issue

partials/
  cadReminder/
    index.ts 
    index.stories.tsx
    index.scss

css/
  cadReminder/
    index.css
  appBadges/
    index.css
```